scripts/build_dists_archive.sh

```diff
@@ -102,10 +105,14 @@ elif [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
 
 elif [ "$(expr substr $(uname -s) 1 10)" == "MINGW32_NT" ]; then
   # Do something under 32 bits Windows NT platform
-  echo "32 bits Windows NT platform"
+ echo "32 bits Windows NT platform"
+ echo "Unsupported, use installer"
```

Comment:
I would suggest a URL explain why it's not support - ie no 32bit support.

Reply:
I think anything other than OSX or Ubuntu should just say "Platform not supported with this build script."

Reply:
Stripped out the architectures which were not ubuntu or osx.